src/index.js

```diff
@@ -1,21 +1,35 @@
 'use strict';
 
+
```

Comment:
Looks like this is causing an eslint error, failing the Travis build

Reply:
Yes, thanks, fixed in 81a8287